test/errors-spec.js

```diff
@@ -29,7 +29,7 @@ describe("Parser:", function() {
         it("rejects \\sqrt as argument to ^", function() {
             expect`1^\sqrt{2}`.toFailWithParseError(
                    "Got function '\\sqrt' with no arguments as superscript" +
-                   " at position 2: 1^̲\\sqrt{2}");
+                   " at position 3: 1^\\̲s̲q̲r̲t̲{2}");
```

Comment:
Why did the position here change from the `\` to the `s`? I guess other errors already pointed there.

Reply:
@edemaine Actually the position is 1-based, and it has changed from the `^` to the `\`, and I think it makes more sense.